src/webapp/src/client/app/lotteries/lotteries-widget/lottery-widget.html

```diff
@@ -0,0 +1,20 @@
+<div id="lotteries-wall-widget">
+    <div class="lottery-activity-log-container" ng-repeat="lottery in latestLotteries"
+        ace-lotteries-detail-modal="lottery.id">
+        <div class="lottery-activity-log-inner">
+            <div class="lottery-activity-log-content-box">
+                <div class="lottery-activity-log-title">
+                    {{ lottery.title }}
+                </div>
```

Comment:
A suggestion: you could display the amount of tickets the user has already bought.